README.md

```diff
@@ -265,14 +265,14 @@ In some cases HMR (hot module reloading) is not working out of the box in this c
 1. In the folder `public-ui-components/packages/{workspace}/forms` run
 
     ```sh
-    yarn unlink
+    pnpm remove @kong-ui-public/forms -g
     ```
 
 1. In the root folder of the host application/package run:
 
     ```sh
-    yarn unlink "@kong-ui-public/forms"
-    yarn install --force --frozen-lockfile
+    pnpm unlink -g @kong-ui-public/forms
+    pnpm install --force --frozen-lockfile
```

Comment:
You shouldn't need to force the reinstall
```suggestion
    pnpm install
```

Reply:
Good point, i've deleted those options.